test/e2e/README.md

```diff
@@ -1,6 +1,59 @@
-# End-to-End Tests
+# Kubestellar end to end testing
+
+This e2e tests can test either (a) the  local copy of the repo or (b) the release identified in the kubestellar PostCreateHook (which will be the last release created, regardless of quality, except for that brief time when it identifies the release about to be made). Testing the local copy is the default behavior; to test the release identified in the PostCreateHook, pass `--released` on the command line of `run-test.sh`.
+
+The kubestellar controller-manager will be invoked with `-v=2` unless otherwise specified on the command line with `--kubestellar-controller-manager-verbosity $number`. This verbosity can not be set to a value other than 2 when using `--released`.
+
+The transport controller will be invoked with `-v=4` unless othewise specified on the command line with `--transport-controller-verbosity $number`.
+
+## Running the test on three new Kind clusters
 
 **PRE-REQ**: All of these tests use three `kind` clusters, so you need to [raise the relevant OS limits](https://kind.sigs.k8s.io/docs/user/known-issues/#pod-errors-due-to-too-many-open-files).
```

Comment:
These tests can now be run on OCP clusters, right? The pre-req statement needs to be qualified to say that the OS limit needs to be raised IF the test will be run on `kind` clusters.

Reply:
Note that the pre-req statement is already inside a section referring to run tests on kind cluster only.

Reply:
Oh, right.